src/components/Dapp.re

```diff
@@ -237,25 +241,52 @@ module GorillaInfo = {
     // https://rimble.consensys.design/components/rimble-ui/EthAddress#props
     // https://github.com/ConsenSys/rimble-ui/blob/dd470f00374a05c860b558a2cb9317861e4a0d89/src/EthAddress/index.js (maybe make a PR here with some changes)
     <Rimble.Box m=5>
-      <p>
+      <div>
         <small>
           <strong>
-            <S> {"Monthly Pledge (at " ++ monthlyRate ++ "%):"} </S>
+            <S> {"Monthly Pledge (at " ++ monthlyRate ++ "%): "} </S>
+            <Rimble.Tooltip
+              message={
+                "This is the monthly percentage contribution of "
+                ++ Gorilla.getName(gorilla)
+                ++ "'s sale price that will go towards conservation of endangered animals. This is deducted continuously from the deposit and paid by the owner of the animal"
+              }
+              placement="top">
+              <span> <S> {js|ⓘ|js} </S> </span>
+            </Rimble.Tooltip>
           </strong>
         </small>
         <br />
         <S> {monthlyPledgeEth ++ " ETH"} </S>
         <br />
         <small> <S> {"(" ++ monthlyPledgeUsd ++ " USD)"} </S> </small>
-      </p>
+      </div>
       <p>
-        <small> <strong> <S> "Current Patron:" </S> </strong> </small>
+        <small>
+          <strong>
+            <S> "Current Patron: " </S>
+            <Rimble.Tooltip
+              message="This is the address of the current owner (crypto wallet 'account' number)"
+              placement="top">
```

Comment:
Maybe instead of crypto wallet 'account' number - we just say--- "This is the public address of the current owner" - think crypto wallet 'account' number sounds more understandble but wallet and crypto etc.. all can have very different meanings. Also think lets stick away from the word crypto - negative sentiment I reckon.